### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by daisycrego**
## Description
Update the README to include more description of the overall system and to link to the frontend documentation, so that the user knows where to look for various docs during the set up process.

## Types of Changes
_Put an `x` in the boxes that apply_

- [ ] Feature (non-breaking change which adds functionality)
- [ ] Bug Fix (non-breaking change that fixes an issue)
- [ ] Breaking Change (feature/fix that causes existing features to not work as expected)
- [x] Documentation

## Checklist
- [x] I have read the [contribute]contributing<link> doc
- [x] Classes, scripts, and environment variables follow existing naming convention
- [x] Lint and Unit tests pass locally
- [x] New features on hardware have been tested on a local Raspberry Pi
- [x] Mention new programs/binaries if any must be installed along with this change
- [x] Mention new environment variables if any have been added to hardware/env file
- [x] Please make sure test coverage does not drop. If it does, please explain the reasons.
- [x] Any new required python modules are added to the requirements.txt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,9 +131,9 @@ You should now be able to view the site at http://127.0.0.1:8000/
 python manage.py test
 ```
 
-# HOWTO Contribute to this repo
+# How to contribute to this repo
 
-Refer to our contributing guidelines [here.](https://github.com/mercury-telemetry/mercury-telemetry/blob/master/CONTRIBUTING.md)
+Refer to our contributing guidelines [here.](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/blob/master/CONTRIBUTING.md)
```
Comment (VentusXu09) [inline README.md]: I think we should use the Contribute file in this new repo

File: README.md [hunk 2]
```diff
@@ -87,7 +117,7 @@ env LDFLAGS="-I/usr/local/opt/openssl/include -L/usr/local/opt/openssl/lib" pip
...
 
-To setup SymmetricsDs on your machine, click [here.](https://github.com/mercury-telemetry/mercury-telemetry/edit/master/README.md) In case you are on windows, please also refer to the wiki on the topic by going [here.](https://github.com/mercury-telemetry/mercury-telemetry/wiki/Setting-up-SymmetricDS-on-Windows) Windows support is not streamlined and is very error prone but this should be enough to get you started. In case you want to help us make this better please refer below as to how you can contribute to the project.
+To setup SymmetricDS on your machine, click [here.](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/blob/master/symmetricds/README.md) In case you are on windows, please also refer to the wiki on the topic by going [here.](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/wiki/Setting-up-SymmetricDS-on-Windows) Windows support is not streamlined and is very error prone but this should be enough to get you started. In case you want to help us make this better please refer below as to how you can contribute to the project.
```
Comment (VentusXu09) [inline README.md]: URLs in this section are still the old repo, please update

File: README.md [hunk 3]
```diff
@@ -48,20 +78,20 @@ env LDFLAGS="-I/usr/local/opt/openssl/include -L/usr/local/opt/openssl/lib" pip
...
 
-4. Install the follwoing dependencies:
-    - [PostgreSQL Setup Guide](https://github.com/mercury-telemetry/mercury-telemetry/wiki/PostgreSQL-Setup-Guide)
-    - [Geckodriver Install Instruction](https://github.com/mercury-telemetry/mercury-telemetry/wiki/Geckodriver-Install-instructions)
-    - [Grafana Setup Guide](https://github.com/mercury-telemetry/mercury-telemetry/wiki/Grafana-Set-Up-Guide)
+4. Install the following dependencies:
+    - [PostgreSQL Setup Guide](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/wiki/PostgreSQL-Setup-Guide)
+    - [Geckodriver Install Instruction](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/wiki/Geckodriver---Install-instructions)
+    - [Grafana Setup Guide](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/wiki/Grafana-Set-Up-Guide)
```
Comment (VentusXu09) [inline README.md]: URLs in this section are still the old repo, please update

File: README.md [hunk 4]
```diff
@@ -26,16 +56,16 @@ sudo apt install python3-dev libpq-dev
...
 
-2. Install the follwoing dependencies:
+2. Install the following dependencies:
 
-    - [PostgreSQL Setup Guide](https://github.com/mercury-telemetry/mercury-telemetry/wiki/PostgreSQL-Setup-Guide)
-    - [Geckodriver Install Instruction](https://github.com/mercury-telemetry/mercury-telemetry/wiki/Geckodriver-Install-instructions)
-    - [Grafana Setup Guide](https://github.com/mercury-telemetry/mercury-telemetry/wiki/Grafana-Set-Up-Guide)
+    - [PostgreSQL Setup Guide](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/wiki/PostgreSQL-Setup-Guide)
+    - [Geckodriver Install Instruction](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/wiki/Geckodriver---Install-instructions)
+    - [Grafana Setup Guide](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/wiki/Grafana-Set-Up-Guide)
```
Comment (VentusXu09) [inline README.md]: URLs in this section are still the old repo, please update

File: README.md (hunk 1 above)
Comment (daisycrego) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (daisycrego) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (daisycrego) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (daisycrego) [reply README.md]: done

**Review by VentusXu09 — APPROVED** [review]
LGTM

**Pull request merged**